core/domain/feedback_services.py

```diff
@@ -179,7 +182,7 @@ def get_last_updated_time(exploration_id):
 
     If this exploration has no threads, returns None.
     """
-    threadlist = get_all_threads(exploration_id, False)
+    threadlist = get_displayable_threads(exploration_id, False)
     return max(
```

Comment:
I am not sure what is the has_suggestion,
but shouldn't be here max from all threads regarding of has_suggestion ? 
It means that: max from get_displayable_threads(exploration_id, False) and get_displayable_threads(exploration_id, True)?